things/api/grpc/responses.go

```diff
@@ -7,3 +7,14 @@ type authorizeRes struct {
 	id         string
 	authorized bool
 }
+
+type verifyConnectionsRes struct {
+	Status      string
+	Connections []ConnectionStatus
+}
+
+type ConnectionStatus struct {
```

Comment:
Do we need to this to export struct ? 
Does it can be internal struct `type connectionStatus struct` ?